include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
Since these are user facing header files, we may need to treat those changes as a special case.  I would rather not include osd.h from a user header file if we can help it.  Maybe this section of code is ifdef out for windows.  It can then be defined in the windows osd.h file, so that all internal code that relies of the definition will work